drivers/sensor/wsen_hids/Kconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig HIDS
+	bool "WSEN-HIDS humidity and temperature sensor"
+	depends on I2C || SPI
+	select HAS_WESENSORS
```

Comment:
Add:
```
default y
depends on DT_HAS_WSEN_HIDS_ENABLED
```

Tweak `depends on I2C || SPI` to be:
```
+       depends on (I2C && $(dt_compat_on_bus,$(DT_COMPAT_WSEN_HIDS),i2c)) || \
+                  (SPI && $(dt_compat_on_bus,$(DT_COMPAT_WSEN_HIDS),spi))
```

Reply:
@yath-eiSmart we are transitioning to the code I suggested.  Take a look at:

https://github.com/zephyrproject-rtos/zephyr/pull/48048

Reply:
Ok, there was a "WE_" missing from `DT_COMPAT_WE_WSEN_HIDS` and `DT_HAS_WE_WSEN_HIDS_ENABLED`, that's why it didn't work when I tried it at first.
I've changed it accordingly.

Reply:
Still need to add selects for I2C and SPI